README.md

```diff
@@ -444,6 +444,7 @@ For documentation on the latest development code see the [documentation index][d
 * [newrelic](./plugins/outputs/newrelic)
 * [nsq](./plugins/outputs/nsq)
 * [opentsdb](./plugins/outputs/opentsdb)
+* [otlp](./plugins/outputs/otlp) (OpenTelemetry Protocol)
```

Comment:
Can we please call it `opentelemetry`!? It's much more obvious than the abbreviation...

Reply:
Sure, that makes sense to me.